### DISCUSSION TITLE
Radiosonde Telemetry Data Storage

### DISCUSSION THREAD

**Discussion by mjl166** [dl9rdz/rdz_ttgo_sonde#374] [Q&A]
This may be out of the scope of this project, but I was wondering if it is possible to store the data from the sonde onto the LoRa board  (Many of them have a micro-SD card slot).  I plan on changing the sonde's frequency and launching it on my own HAB with a GoPro and would like to have the flight data so I can display it on the video.
Thanks!

**Comment by dl9rdz** [comment]
Indeed I might implement such feature for a different project some time soon.

There is no space in the current firmware, so such a feature will not be included in the standard firmware image. You will have to compile the code (and either disable other features, or use a large partition scheme (which disabled OTA))

**Comment by mjl166** [comment]
Ok.  Thanks for the info! Sent from Mail for Windows From: dl9rdzSent: Sunday, August 6, 2023 4:07 AMTo: dl9rdz/rdz_ttgo_sondeCc: mjl166; AuthorSubject: Re: [dl9rdz/rdz_ttgo_sonde] Radiosonde Telemetry Data Storage (Discussion #374) Indeed I might implement such feature for a different project some time soon.There is no space in the current firmware, so such a feature will not be included in the standard firmware image. You will have to compile the code (and either disable other features, or use a large partition scheme (which disabled OTA))—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
